### DISCUSSION TITLE
Win+Tab in ubuntu 20.04

### DISCUSSION THREAD

**Discussion by R2K95** [iberianpig/fusuma#245] [Q&A]
hi,
I want to know how to achieve switching applications using touch pad, instead of clicking win+tab keys or Alt tab, using fusuma. I am using ubuntu 20.04
I tried with following config file, but didnt work for me. Please comment how achieve my usecase

swipe:
  3: 
    left: 
      command: 'xdotool key alt+Left'
    right: 
      command: 'xdotool key alt+Right'
    up: 
      command: 'xdotool key super+Up'
    down: 
      command: 'xdotool key super+Down'
  4:
    left: 
      command: 'xdotool key super+tab+Left'
    right: 
      command: 'xdotool key super+tab+Right'
    up: 
      command: 'xdotool key ctrl+alt+Down'
    down: 
      command: 'xdotool key ctrl+alt+Up'
pinch:
  in:
    command: 'xdotool keydown ctrl click 4 keyup ctrl'
  out:
    command: 'xdotool keydown ctrl click 5 keyup ctrl'

**Comment by iberianpig** [comment]
Could you try this configuration?
[3 fingers Alt Tab Switcher(Windows Style) · iberianpig/fusuma Wiki](https://github.com/iberianpig/fusuma/wiki/3-fingers-Alt-Tab-Switcher(Windows-Style))

**Reply by R2K95** [reply]
@iberianpig  That shown the windows currently open in device. But the working I needed is to switch to next application/window when I swipe, like in macos devices. Here it moves between the pop up showing the windows but need an extra click to select. I want to avoid that click

Also after swipe and select, my touch pad shows discrepancy in its working, drag and click not work correctly after that.
I am using;
Fusuma: 2.0.5
libinput: 1.15.5
ruby 2.7.0p0
